### DISCUSSION TITLE
Is there a json schema somewhere for ReverseProxy configuration

### DISCUSSION THREAD

**Discussion by aDisplayName** [dotnet/yarp#1856] [Ideas] [answered]
Can we have a json schema somewhere to reference to, to help with writing the ReverseProxy configuration?

**Comment by MihaZupan** [comment, marked as answer]
We don't have one yet, but we recognize the value and are tracking adding it in https://github.com/microsoft/reverse-proxy/issues/1350

**Reply by Tratcher** [reply]
Also see this doc: https://microsoft.github.io/reverse-proxy/articles/config-files.html
